JetNews/app/src/main/java/com/example/jetnews/utils/WindowSize.kt

```diff
@@ -39,50 +38,44 @@ import androidx.window.layout.WindowMetricsCalculator
  */
 enum class WindowSize { Compact, Medium, Expanded }
 
+/**
+ * Partitions a [DpSize] into a enumerated [WindowSize] class.
+ */
 @VisibleForTesting
-fun getWindowSize(width: Dp): WindowSize = when {
-    width.value < 0f -> throw IllegalArgumentException("Dp value cannot be negative")
-    width.value < 600f -> WindowSize.Compact
-    width.value < 840f -> WindowSize.Medium
+fun getWindowSizeClass(windowDpSize: DpSize): WindowSize = when {
```

Comment:
This function shouldn't be annotated with `@VisibleForTesting`, right? It's used JetsnackApp

Reply:
I assume you mean `JetNewsApp`?

The only place it is used is in `WindowSize`, so `@VisibleForTesting` is correct.

Reply:
Ah ok ok! Makes sense, Thanks for moving it down